packages/types/lib/driver.ts

```diff
@@ -2200,21 +2198,21 @@ export interface ExternalDriver<C extends Constraints = BaseDriverCapConstraints
  *
  * This is likely unusable by external consumers, but YMMV!
  */
-export interface DriverStatic<D extends Driver> {
+export interface DriverStatic<T extends Driver> {
   baseVersion: string;
   updateServer?: UpdateServerCallback;
-  newMethodMap?: MethodMap<D>;
-  executeMethodMap?: ExecuteMethodMap<D>;
+  newMethodMap?: MethodMap<T>;
+  executeMethodMap?: ExecuteMethodMap<T>;
 }
 
 /**
  * Represents a driver class, which is used internally by Appium.
  *
  * This is likely unusable by external consumers, but YMMV!
  */
-export type DriverClass<D extends Driver = ExternalDriver> = Class<
-  D,
-  DriverStatic<D>,
+export type DriverClass<T extends Driver = Driver> = Class<
```

Comment:
while `ExternalDriver` may be more _likely_, it is not necessarily an `ExternalDriver`.